apps/etterlatte-oppdater-behandling/src/main/kotlin/TidshendelseRiver.kt

```diff
@@ -88,15 +75,17 @@ class TidshendelseRiver(
                                 merknad = "Aldersovergang",
                                 frist = frist,
                             )
-                        kvittering[HENDELSE_DATA_KEY] = oppgaveId
+                        hendelseData["opprettetOppgaveId"] = oppgaveId
                     } else {
                         logger.info("Dry run: skipper oppgave")
                     }
                 } else {
                     logger.info("Ingen løpende ytelse funnet for sak $sakId")
                 }
 
-                context.publish(kvittering.toJson())
+                packet[ALDERSOVERGANG_STEG_KEY] = "OPPGAVE_OPPRETTET"
```

Comment:
Har vi ein grunn til å sende vidare viss det ikkje er løpende ytelse?

Reply:
Nei, det er egentlig bare for å ha noe som `HendelseRiver` (i tidshendelser-appen) kan lytte på å markere en hendelse som "ferdig" der, nå er jo det "OPPGAVE_OPPRETTET". Bør finne et generelt navn som indikerer at hendelsen er ferdig behandlet.